.github/workflows/rolling.yaml

```diff
@@ -0,0 +1,44 @@
+name: rolling
+
+on:
+  pull_request:
+    branches:
+      - rolling
+      - rep-mocap
+  push:
+    branches:
+      - rolling
+      - rep-mocap
+
+jobs:
+  build-and-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-24.04]
+      fail-fast: false
+    steps:
+      - name: Setup direc tories
+        run: mkdir -p ros_ws/src
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
```

Comment:
I am not sure that you need to specify this. Maybe that's the reason why it is not finding the file in CI 🤔

Reply:
Does v3 exist? I ask from my ignorance, since I always use v2.

And why does it create the workspace?

Reply:
Yes, I have the same questions 🤔

Reply:
Yes, even v4: https://github.com/actions/checkout/tags

Reply:
Oh, how funny, we're using a version from a year ago

Reply:
> I am not sure that you need to specify this. Maybe that's the reason why it is not finding the file in CI 🤔

I have changed it, Let's see if it works.